src/capwap_ac_gtk_rekey.erl

```diff
@@ -1,4 +1,4 @@
-%% Copyright (C) 2017, Travelping GmbH <info@travelping.com>
+%% Copyright (C) 2013-2023, Travelping GmbH <info@travelping.com>
```

Comment:
There is not need to update the copyright year on existing files

Reply:
ok